app/src/main/java/ar/com/wolox/android/example/ui/news/NewsViewHolderAdapter.kt

```diff
@@ -0,0 +1,80 @@
+package ar.com.wolox.android.example.ui.news
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageButton
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.model.News
+
+class NewsViewHolderAdapter : RecyclerView.Adapter<NewsViewHolderAdapter.ViewHolder>() {
```

Comment:
This is OK. However, we are going to extend `NewsViewHolderAdapter` from `ListAdapter`

Reply:
What happened with this?